skietbaan_fe/src/components/ScoreCapture.js

```diff
@@ -279,94 +295,146 @@ export default class search extends Component {
 
   }
 
+
+  goBack() {
+    this.setState({
+      ImageTaken: false,
+      showCamera: false
+    });
+    let video = document.getElementById('video');
+    video.pause();
+    this.ToggleNavbar();
+  }
+
+
   render() {
     let competitionItem = [];
     if (this.state.competitionsList && this.state.competitionsList.length > 0) {
       for (let i = 0; i < this.state.competitionsList.length; i++) {
-        competitionItem.push(<div className="competition-item-container"> <div key={'mykey' + i} 
-        className={this.state.clicked != null && this.state.clicked === i ? "active competition-item" : "competition-item"}>
-        <li onClick={() =>this.CompetitionClicked(i, this.state.competitionsList[i].name)}>
-          {this.state.competitionsList[i].name} </li></div></div>);
+        competitionItem.push(<div className="competition-item-container"> <div key={'mykey' + i}
+          className={this.state.clicked != null && this.state.clicked === i ? "active competition-item" : "competition-item"}>
+          <li onClick={() => this.CompetitionClicked(i, this.state.competitionsList[i].name)}>
+            {this.state.competitionsList[i].name} </li></div></div>);
 
       }
     }
 
     return (
       <div className="position-relative">
-        <div className={this.state.showCamera && !this.state.ImageTaken ? "back-white page-content no-padding" :
-          this.state.ImageTaken ? "opacity-85 page-content no-padding" : "page-content no-padding"}>
-          <div className={this.state.showCamera && !this.state.ImageTaken ? "hidden" : ""}>>
-        <div className="label-score">
-              <label>Type in score</label>
+        <div className={this.state.showCamera || this.state.ImageTaken ? "" : "white-border"}></div>
```

Comment:
Extract this comparison as it is done three times

Reply:
done